.github/workflows/publish_master.yml

```diff
@@ -70,6 +70,51 @@ jobs:
           context: .
           file: ./Dockerfile
           platforms: linux/amd64,linux/arm64
+          run: |
+            pip install -r requirements.txt
+            docker run -d -p 48008:8080 ${{ secrets.AWS_ECR_DEV_REPO }}
```

Comment:
This workflow should build the image from the master branch, not run the image from the dev repo.